src/exporter.js

```diff
@@ -124,6 +125,10 @@ export default class {
   }
 }
 
+export const SPACE_REPLACER = String.fromCharCode(160);
```

Comment:
Replace it to underscore.

- The original Ruby gem [replaces](https://github.com/albertzak/anki2/blob/5eb49e82309457cf24e1469535784c5ca900c639/lib/anki2.rb#L98) it to underscore characters.

- http://ankisrs.net/docs/manual.html uses same convention 
![image](https://cloud.githubusercontent.com/assets/1713789/21081432/96b58fd4-c001-11e6-848a-d720a2fd40ad.png)

I suppose it is quite possible that Anki clients have logic to transform `underscore_case` to `Human Case` on searching / displaying tags.